20180321/README.md

```diff
@@ -1 +1,188 @@
-# From 0 to Match - Set Up Everything with Apple Xcode
+# First mobile app with Matchmore
+
+In this post, we will go through setting up an iOS application with AlpsSDK, from registration on the portal to obtaining a match.
+
+## Before you start
+We will use cocoapods, to install it execute in terminal:
+```
+sudo gem install cocoapods
+```
+
+## Get API key
+Now let's create a new application on developer portal and get API key that we will later use in SDK.
+After loging in (you can register for free) you do:
+
+![alt text](https://raw.githubusercontent.com/matchmore/tech-blog/master/20180321/img/create-app.gif "create app")
+
+## Xcode mobile app
+Now we create new iOS single view application in Xcode (select swift as language) after that close Xcode and go to main application directory with terminal (in this case my-app). Execute `pod init` which will create `Podfile` file. To add AlpsSDK to application, add `pod 'AlpsSDK', '~> 0.6'` to `Podfile` 
+
+After the edit, it may look like this
+```
+target 'my-app' do
+  # Comment the next line if you're not using Swift ...
+  use_frameworks!
+
+  pod 'AlpsSDK', '~> 0.6'
+  ...
+```
+
+After that execute `pod install`, this will download AlpsSDK and generate `my-app.xcworkspace` that you have to import in Xcode (the old my-app.xcodeproj will not work correctly).
+
+After importing `my-app.xcworkspace` into Xcode, we will add necessary permissions for the app in `info.plist` file. Right click on `info.plist` select `Open As > Source Code` and add two keys at the end:
+
+```
+...
+    <key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+    <string>This app needs your current location</string>
+    
+    <key>NSLocationWhenInUseUsageDescription</key>
+    <string>This app needs your current location</string>
+</dict>
+</plist>
+```
+
+## SDK configuration
+Next we have to initialize SDK. To do that in `AppDelegate.swift` in `application` funtion add thos two lines:
```

Comment:
usually you write function names like this:
`application:didFinishLaunchingWithOptions()`

Reply:
there're like 5 functions that start with `application keyword`